utilities/convert_frosst/smirffishFrcmod.parm99Frosst.txt

```diff
@@ -361,14 +363,15 @@ DIHE
 #
 # C-O bonds
 # O-Csp3
-[*:1]-[#6X4:2]-[#8X2H1:3]-[#1:4]     1  0.160    0.0  -3.0  parm99 X_-CT-OH-X  JCC,7,(1986),230
-[*:1]-[#6X4:2]-[#8X2H1:3]-[#1:4]     1  0.250    0.0   1.0  parm99 HO-OH-CT-CT  Junmei et al, 1999
-[*:1]-[#6X4:2]-[#8X2H0:3]-[*:4]      1  0.383    0.0  -3.0  parm99 CT-CT-OS-CT 
-[*:1]-[#6X4:2]-[#8X2H0:3]-[*:4]      1  0.100  180.0   2.0  parm99 CT-CT-OS-CT 
```

Comment:
@davidlmobley it looks like you're removing the generic `*-Csp3-O-*` torsions. Is the reasoning here that the parm99 numbers were for CT-CT-OS-CT are we sure Christopher didn't combine anything into those generics?